docs/remote_inference_blueprints/aleph_alpha_connector_luminous_base_embedding_blueprint.md

```diff
@@ -0,0 +1,143 @@
+# Aleph Alpha connector blueprint example for luminous-base embedding model
+
+## 1. Add connector endpoint to trusted URLs:
+
+Note: no need to do this after 2.11.0
```

Comment:
I this is copied from other doc, remove this one  as we always need to update this setting for this model